app/Jobs/CreateQueryserviceBatchesJob.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace App\Jobs;
+
+use App\EventPageUpdate;
+use App\QsBatch;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Config;
+
+class CreateQueryserviceBatchesJob extends Job
+{
+    private const NAMESPACE_ITEM = 120;
+    private const NAMESPACE_PROPERTY = 122;
+    private const NAMESPACE_LEXEME = 146;
+
+    private int $entityLimit;
+
+    public function __construct()
+    {
+        $this->entityLimit = Config::get('wbstack.qs_batch_entity_limit');
+    }
+
+    public function handle(): void
+    {
+        DB::transaction(function () {
+            // Get the latest batch that we have created (or 0)
+            $maxIdBatch = QsBatch::orderBy('eventTo', 'desc')->first();
```

Comment:
This works and is easy to understand, but has one terrible drawback: it makes it very hard for us to manually rewind and play back things that we missed. I start wondering if we should really just persist the value in the database, even if it means creating a table with a single column and a single row.

Reply:
So we think that actually we do want a single column for this after much thinking